content/en/blog/2024/sig-arch-code-spotlight.md

```diff
@@ -0,0 +1,138 @@
+companies, must also be a challenge: how does the subproject operate in terms of making sure that
+standards are being followed?
+
+**MJ**: When it comes to dependency management in the project, there is a [dedicated
+team](https://github.com/kubernetes/org/blob/a106af09b8c345c301d072bfb7106b309c0ad8e9/config/kubernetes/org.yaml#L1329)
+that helps review and approve dependency changes. These are folks who have helped lay the foundation
+of much of the
+[tooling](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/vendor.md)
+that Kubernetes uses today for dependency management. This tooling helps ensure there is a
+consistent way that contributors can make changes to dependencies. The project has also worked on
+additional tooling to signal statistics of dependencies that is being added or removed:
+[`depstat`](https://github.com/kubernetes-sigs/depstat)
+
+Apart from dependency management, another crucial task that the project does is management of the
+staging repositories. The tooling for achieving this (`publishing-bot`) is completely transparent to
+contributors and helps ensure that the staging repos get a consistent view of contributions that are
+submitted to `kubernetes/kubernetes`.
+
+Code Organization also works towards making sure that Kubernetes [stays on supported versions of
+Go](https://github.com/kubernetes/enhancements/tree/cf6ee34e37f00d838872d368ec66d7a0b40ee4e6/keps/sig-release/3744-stay-on-supported-go-versions). The
+linked KEP provides more context on why we need to do this. We collaborate with SIG Release to
+ensure that we are testing Kubernetes as rigorously and as early as we can on Go releases and
+working on changes that break our CI as a part of this. An example of how we track this process can
+be found [here](https://github.com/kubernetes/release/issues/3076).
+
+## Release cycle and current priorities
+
+**FSM**: Is there anything that changes during the release cycle?
+
+**MJ** During the release cycle, specifically before code freeze, there are often changes that go in
+that add/update/delete dependencies, fix code that needs fixing as part of our effort to stay on
+supported versions of Go.
+
+Furthermore, some of these changes are also candidates for
+[backporting](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md)
+to our supported release branches.
+
+**FSM**: Is there any major project or theme the subproject is working on right now that you would
+like to highlight?
+
+**MJ**: I think one very interesting and immensely useful change that has been recently added (and I
+take the opportunity to specifically highlight the work of [Tim Hockin](https://github.com/thockin)
+on this) is the introduction of [Go workspaces to the Kubernetes
+repo](https://github.com/kubernetes/enhancements/tree/master/keps/sig-architecture/4402-go-workspaces). A
```

Comment:
Could instead link to [`/blog/2024/03/19/go-workspaces-in-kubernetes/`](https://k8s.dev/blog/2024/03/19/go-workspaces-in-kubernetes/), the article about this.

Reply:
Yes much better. Changed in last commit.